src/Resources/Resources.Test/Resources.Test.csproj

```diff
@@ -21,6 +21,7 @@
   </ItemGroup>
 
   <ItemGroup>
+    <ProjectReference Include="..\ResourceManager\ResourceManager.csproj" />
```

Comment:
Added this reference to be able to access JsonExtensions, probably, as a future PR, consider moving JsonExtensions to a common project.